resources/views/community/components/forum/post-comment-meta.blade.php

```diff
@@ -40,6 +52,23 @@
 @endif
 
 <p class='smalltext !leading-[14px]'>
-    {{-- Keep this all on a single line so white space isn't added before the comma --}}
-    {{ $formattedPostTimestamp }}@if($formattedEditTimestamp)<span class='italic smalltext !leading-[14px]'>, <span class='hidden sm:inline'>last</span> edited {{ $formattedEditTimestamp }}</span> @endif
+    <span 
+        title="{{ $shouldUsePostedTimeAgoDate ? $postCreatedTimestamp : null }}" 
```

Comment:
Would it be possible to use the pretty-formatted date in to the tooltip?

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/9ded8ff1-25c9-4667-b18f-70b185c65fd8)

i.e. `July 14 2023, 6:36pm`

Reply:
Same comment for forum lists.

Reply:
Good call, resolved in https://github.com/RetroAchievements/RAWeb/pull/1652/commits/ae22870247e6e877641fa493b7570f0a74d1d5d8.

Reply:
It's still inconsistent between the tooltip and what's displayed in the text:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/6f1b6ded-30e4-4251-aade-8580180bf4bc)

"July 12" vs "15 Jul" and "5:22pm" vs "15:37"

Reply:
Ah sorry, I went a little too fast and didn't notice `getNiceDate()` outputted a different format. Should be cleared up in https://github.com/RetroAchievements/RAWeb/pull/1652/commits/86d2636d70f23ee2df3f77528d58a939aeb94e2f.